Dockerfile

```diff
@@ -0,0 +1,65 @@
+# Build:
+#   $ docker build -t yottadb:latest .
+# Use with data persistence:
+#   $ docker run --rm -e gtm_chset=utf-8 -v `pwd`/ydb-data:/data -ti yottadb:latest
+
+# Stage 1: YottaDB build image
+FROM ubuntu as ydb-release-builder
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update \
+ && apt-get install -y \
+      file \
+      cmake \
+      tcsh \
+      libconfig-dev \
+      libelf-dev \
+      libgcrypt-dev \
+      libgpg-error-dev \
+      libgpgme11-dev \
+      libicu-dev \
+      libncurses-dev \
+      libssl-dev \
+      zlib1g-dev \
+ && apt-get clean
+
+ADD . /tmp/yottadb-src
```

Comment:
Once this change gets committed, please build a r1.10-specific (not current master) docker image and ship that to Dockerhub. Thanks.

Reply:
BTW, the image built out of my original PR is there already https://hub.docker.com/r/ideasys/yottadb. Can we just add this docker build and push process to your existing CI environment using your YottaDB DockerHub organization to keep it all originating from you? :)

Reply:
The plan is to have docker images as part of the release process for new versions of yottadb and we already have a dockerhub organization to do just what you say :)

Reply:
Cool! :-)